src/analytics/private/model.ts

```diff
@@ -211,6 +211,10 @@ export type MustacheParamType = {
   table_clickstream_log: string;
   table_refresh_mv_sp_status: string;
   table_ods_events: string;
```

Comment:
clean the legacy tables

Reply:
sp-migrate-ods-events-1.0-to-1.1.sql still use it.

Reply:
refactored it